### PR TITLE
.coafile: Replace Default section with all

### LINKED ISSUE #2397: .coafile: Replace default section with all
Since **Default** section is deprecated, it should be replaced with **all**.
Many people might take `.coafile` of coala as a reference for their project, so IMO it should be updated.

### PR DESCRIPTION
**Pull request by rammanoj**
As Default section is deprecated, it is replaced with all in .coafile.

Closes #2397

### CODE REVIEW

**Comment by rohanrk** [comment]
Next time don't close your PR. Just force push after you squash your commits.

File: .coafile [hunk 1]
```diff
@@ -44,16 +44,16 @@ body_close_issue_full_url = True
...
 body_enforce_issue_reference = False
 
-[LineCounting]
+[all.LineCounting]
 enabled = False
 bears = LineCountBear
 max_lines_per_file = 1000
 
-[rst]
+[all.rst]
```
Comment (MacBox7) [inline .coafile]: IMO this section can be independent and doesn't need to be inherited from all since you are replacing the files.

**Comment by rammanoj** [comment]
@MacBox7 thanks for the review. I committed it again. Can you review it now once? I guess the PR solves the issue.

**Comment by Vamshi99** [comment]
@rammanoj You didn't change the rst section yet

**Comment by ishanSrt** [comment]
@Vamshi99 as suggested by @MacBox7 it doesn't really need to derive from the `[all]` section. @rammanoj can you also check whether other sections are also there that don't actually require to be derived from `[all]` ?

**Comment by rammanoj** [comment]
thanks, @ishanSrt I guess no others need to be modified. What is your opinion @MacBox7?

**Comment by MacBox7** [comment]
There is at least 1 more, I was hoping you would figure it out on your own. Because the first newcomer task is running coala on your project itself :wink:.

**Comment by ishanSrt** [comment]
technically git section also shouldn't require the files from `[all]` but waiting for a PR to be merged, will reference this PR there if it gets merged before this, then you can make changes. 👍

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2402/commits/633a46dde4c995c683e6819c4daecc1a07672502

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**